docs/operations-guide/environment-variables.md

```diff
@@ -1,61 +1,810 @@
-| MB_SITE_NAME                               | string                                   | `"Metabase"`                                                   | The name used for this instance of Metabase.                                                                                                                                                                                                                                                               |
-| MB_SITE_URL                                | string                                   | `null`                                                         | The base URL of this Metabase instance, e.g. "http://metabase.my-company.com".                                                                                                                                                                                                                             |
-| MB_SLACK_TOKEN                             | string                                   | `null`                                                         | Slack API bearer token obtained from https://api.slack.com/web#authentication                                                                                                                                                                                                                              |
-| MB_SOURCE_ADDRESS_HEADER                   | string                                   | `X-Forwarded-For`                                              | Identify the source of HTTP requests by this headers value, instead of its remote address.                                                                                                                                                                                                                 |
-| MB_SSL_CERTIFICATE_PUBLIC_KEY              | string                                   | `null`                                                         | Base-64 encoded public key for this sites SSL certificate.Specify this to enable HTTP Public Key Pinning.See http://mzl.la/1EnfqBf for more information.                                                                                                                                                   |
+Many settings in Metabase can be viewed and modified in the Admin Panel, or set via environment variables. The environment variables always take precedence.
+
+Setting environment variables can be done in various ways depending on how Metabase is being run.
+
+JAR (Note that on Windows, use `set` instead of `export`):
+
+```
+export MB_SITE_NAME="Awesome Company"
+java -jar metabase.jar
+```
+
+JAR alternative (declared with `-D` prepending the variable):
+
+```
+java -DMB_SITE_NAME="Awesome Company" -jar metabase.jar
```

Comment:
shouldn't it be -Dmb.site.name as per https://github.com/metabase/metabase/blob/master/src/metabase/config.clj#L42

Reply:
It can be both, but I find it confusing that we're starting to use two different types of variable - which also makes it more difficult for people to search for questions. See #12475